src/redis.ts

```diff
@@ -87,6 +88,11 @@ export type RedisCacheParameter = {
   GETRedisTimeout?: number;
 
   onError?: (err: unknown) => void;
+
+  /**
+   * @default 10
```

Comment:
is default 10 too low? 🤔

Reply:
Yeah, could do 100

Reply:
I will put default at 20, libraries should have safe defaults